apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -17,10 +21,8 @@ var Instructions = require('./Instructions');
 var CollapserIcon = require('./CollapserIcon');
 var HeightResizer = require('./HeightResizer');
 var msg = require('@cdo/locale');
-import ContainedLevel from '../ContainedLevel';
-import PaneHeader, { PaneButton } from '../../templates/PaneHeader';
-import experiments from '@cdo/apps/util/experiments';
 
+var showVideoDialog = require('../../code-studio/videos').showVideoDialog;
```

Comment:
Preference for import over require in new code. (i.e. import { showVideoDialog} from ' ...')